config/manager/kustomization.yaml

```diff
@@ -13,5 +13,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: localhost:5000/profiles-controller
-  newTag: latest
+  newName: skarlso/profiles-controller
```

Comment:
🤔

Reply:
What the heck... I swear I committed the fix to this at least twice. :D Something must be off from rebasing.

Reply:
Thanks for spotting it. I had to do a lot of testing yesterday on my own fork until this all worked. :)